vNext/AISKU/snippet/snippet.js

```diff
@@ -1,7 +1,5 @@
 ﻿(function() {
-    var sdkInstance = "appInsightsSDK";
-    window[sdkInstance] = "appInsights";
-    var aiName = window[sdkInstance]; // provide non default instance name through key appInsightsSDK
+    var aiName = "appInsights";
```

Comment:
With this test, can you test that using 2 snippets, each with different `namePrefix`, on the same page works correctly?

Reply:
Yes everything works correctly